.github/workflows/ci-bump-version.yml

```diff
@@ -36,21 +36,19 @@ jobs:
       - name: Install Poetry
         uses: Gr1N/setup-poetry@v8
 
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
       - name: Build core library
-        working-directory: ./lib/core
-        run: npm ci && npm run build
+        run: pnpm --filter @common-grants/core run build
```

Comment:
The one other pro of moving all of our GH actions to `pnpm` is that this gets a little cleaner